tip-wlan/charts/cassandra/templates/secret.yaml

```diff
@@ -24,9 +24,9 @@ metadata:
 type: Opaque
 data:
   truststore: {{ .Files.Get "resources/config/certs/truststore.jks" | b64enc }}
-  truststore-password: {{ .Files.Get "resources/config/certs/truststore_creds" | b64enc }}
+  truststore-password: {{ .Values.creds.sslTruststorePassword | b64enc }}
```

Comment:
This is good. Thank you for removing the files and instead getting the values from secrets